sphinx_gallery/tests/test_backreferences.py

```diff
@@ -137,10 +148,10 @@ def test_identify_names2(tmpdir):
 Title
 -----
 
-This example uses :func:`h.i`.
+This example uses :func:`k.l`.
 '''
 """ + code_str.split(b"'''")[-1]
-    expected['h.i'] = {u'module': u'h', u'module_short': u'h', u'name': u'i',
+    expected['k.l'] = {u'module': u'k', u'module_short': u'k', u'name': u'l',
```

Comment:
Can you also check the `tinybuild` and add tests that fail in `master` and work now? It ensures that the real-world / end use cases improve in addition to our lower-level functions.

There might already be examples that didn't work before but now work (i.e., just change `test_full.py` but not anything in the `tinybuild/` directory itself), which would be great. But if you need to also change things in `tinybuild/` to get something that fails in `master` but works here it's fine, too.

Reply:
There isn't an appropriate import example in `tinybuild/` - I can add a `import a.b` to `/plot_numpy_matplotlib` ('Link to other packages') example and add an assertion somewhere in `test_full`. Does that work?

Reply:
Yes that works, but in addition to what you've done you should add something like the following after line ~198 in `test_full.py` as that's where we test that the code blocks have the appropriate outgoing links:
```
assert 'sphinx_gallery.backreferences.identify_names.html' in lines
```
It's fine to have all this in `plot_numpy_matplotlib.py` of `tinybuild`, no need for a separate file.